components/02-elements/rating/rating--summary.hbs

```diff
@@ -1,5 +1,5 @@
-<div class="rating__summary {{ class }}" {{{ attributes }}}>
-    <span class="rating__fill" {{{ percentFilled }}}>
-        {{{ percent }}}
-    </span>
+<div class="rating rating--summary {{ class }}" {{{ attributes }}}>
+    <div class="rating__star {{ star.class }}" {{{ star.attributes }}}>
+        <span class="rating__indicator {{ indicator.class }}" {{{ indicator.attributes }}}></span>
+    </div>
```

Comment:
Recently we were talking about rules order inside styles - if you have some time please check the order of the rules and change it where possible :) (not a bug)

Reply:
changed - but I think aZ with automation would be better solution or by group (but make it do by machine not us :D)